src/envo/envo-edit.owl

```diff
@@ -27653,15 +27653,19 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02500044> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03000000> (névé)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000000> "Young granular snow which has undergone melting refreezing and compaction processes")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000000> "Snow which has undergone several cycles of melting and refreezing while being compacted.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000000> "http://orcid.org/0000-0002-3410-4655")
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000000> "https://en.wikipedia.org/wiki/N%C3%A9v%C3%A9")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000000> "Névé that survives a full season of ablation turns into firn.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000000> "névé"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000000> <http://purl.obolibrary.org/obo/ENVO_01000293>)
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03000002> (firn)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000002> "An intermediate type of ice between snow and glacial ice derives from from compacted and recrystalized névé.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000002> "Ice which is an intermediate between snow and glacial ice and which is formed by the compaction and recrystalisation of snow melted from névé.")
```

Comment:
This definition, like that of névé  doesn't really tell me much about the properties of the ice itself. Also, we need to work on "glacial ice". Please open issues on these defs.